module_utils/argspec/base.py

```diff
@@ -0,0 +1,24 @@
+from ansible.module_utils.six import iteritems
+
+
+class ArgspecBase(object):
+
+    argument_spec = {}
+
+    def __init__(self, **kwargs):
+        self.values = {}
```

Comment:
The instance variable _should_ be named `self._values` imho to denote its an internal implementation